src/content/docs/installation/installation_t2macbook.mdx

```diff
@@ -3,98 +3,120 @@ title: T2 MacBook Installation
 
-CachyOS provides out of the box support for the T2 MacBook. The required kernel patches are shipped in all the kernels provided by CachyOS.
+If you still decide to remove macOS, ensure you have a full backup and create a [bootable macOS installer](https://support.apple.com/en-us/101578) beforehand.
+:::
 
-**WiFi connection** is **not available** out of the box because it is a closed source firmware, which CachyOS is not allowed to redistribute.
-The installation needs internet to proceed, so either use a LAN network, or follow [this section](#getting-wifi-firmware-for-the-installation) to get WiFi firmware.
+:::note[Wi-Fi Firmware Required]
+Wi-Fi **will not work** out of the box during or after installation because the required firmware is proprietary and cannot be redistributed by CachyOS. You will need an internet connection (e.g., Ethernet, USB tethering) for the installation process unless you follow the steps below to prepare the Wi-Fi firmware beforehand and enable it during installation.
+:::
 
-### Getting WiFi firmware for the installation
+## Creating a macOS Recovery Installer
 
-<Steps>
-
-1. First boot into macOS and run this command:
-    ```sh
-    curl -sL https://wiki.t2linux.org/tools/firmware.sh | bash -s copy_to_efi
-    ```
-
-2. Now boot into the CachyOS ISO. Open Konsole there, and run these commands:
-    ```sh
-    sudo mkdir -p /tmp/apple-wifi-efi
-    sudo mount /dev/nvme0n1p1 /tmp/apple-wifi-efi
-    bash /tmp/apple-wifi-efi/firmware.sh get_from_efi
-    sudo umount /tmp/apple-wifi-efi
-    ```
-
-3. Setup `iwd` as backend for NetworkManager. You can run the following commands in Konsole to do so:
-    ```sh
-    cat <<EOF | sudo tee /etc/NetworkManager/conf.d/wifi_backend.conf
-    [device]
-    wifi.backend=iwd
-    EOF
-    sudo systemctl restart NetworkManager
-    ```
+It is highly recommended to create a bootable macOS installer **before** you modify your partitions or install CachyOS, especially if you are considering removing macOS entirely. This USB installer allows you to reinstall macOS for firmware updates, troubleshooting, or restoring your system if needed.
 
+<Steps>
+1.  **Obtain a USB Drive:** You'll need an external USB drive (at least 16GB, ideally 32GB).
+2.  **Follow Apple's Guide:** Apple provides detailed instructions on how to download a macOS installer and create a bootable USB drive from it.
+    *   **Follow the official guide here:** [How to create a bootable installer for macOS - Apple Support](https://support.apple.com/en-us/101578)
+3.  **Store Safely:** Keep this USB drive in a safe place in case you need to restore macOS later.
 </Steps>
 
-Now you can connect to a WiFi network and proceed with the installation.
+## Pre-Installation
 
-### Proceeding with Installation
+Perform these steps within macOS **before** booting the CachyOS installer. Ensure you have created your [macOS Recovery Installer](#creating-a-macos-recovery-installer) if desired.
```

Comment:
No need to ensure macOS recovery installer. It's practically not feasible by many.